azure-pipelines/pull-request-validation/build-consumers.yml

```diff
@@ -194,3 +199,75 @@ stages:
       displayName: Run adal Unit tests
       inputs:
         tasks: adal:testLocalDebugUnitTest -Plabtest -ProbolectricSdkVersion=${{variables.robolectricSdkVersion}}
+  # OneAuth
+  - job: oneAuthValidation
+    displayName: OneAuth
+    dependsOn: setupBranch
+    variables:
+      commonBranch: $[ dependencies.setupBranch.outputs['setvarStep.commonBranch'] ]  # map in the variable
```

Comment:
is this different from the "commonBranch" variable that is already available?

Reply:
no its the same, we are just setting it to make use of it